core-bundle/src/Util/LocaleUtil.php

```diff
@@ -56,8 +56,12 @@ public static function getFallbacks(string $locale): array
             return self::$fallbacks[$locale];
         }
 
+        $result = [];
         $data = \Locale::parseLocale($locale);
-        $result = [$data[\Locale::LANG_TAG]];
+
+        if (isset($data[\Locale::LANG_TAG])) {
+            $result[] = $data[\Locale::LANG_TAG];
+        }
```

Comment:
Instead we should add this here:
```suggestion
        } else {
            $data[\Locale::LANG_TAG] = '';
        }
```

Reply:
You mean like this?

```php
$data = \Locale::parseLocale($locale);

if (!isset($data[\Locale::LANG_TAG])) {
    $data[\Locale::LANG_TAG] = '';
}

$result = [$data[\Locale::LANG_TAG]];
```

Reply:
No, if `$data[\Locale::LANG_TAG]` is not set we should not add the empty string to the result array, but we need to set `$data[\Locale::LANG_TAG] = ''` in order for the other candidates to work correctly like `_DE`.